packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -119,12 +119,14 @@ export class AppComponent implements OnInit {
   }
 
   login() {
-    this.oktaAuth.loginRedirect('/profile');
+    this.oktaAuth.signInWithRedirect({
+      originalUri: '/profile'
+    });
   }
 
   async logout() {
     // Terminates the session with Okta and removes current tokens.
-    await this.oktaAuth.logout();
+    await this.oktaAuth.signOut();
```

Comment:
I believe this function will perform a full redirect to Okta and back to the application. So there may be no need for the router navigateByUrl